config.toml

```diff
@@ -100,6 +100,12 @@ privacy_policy = "https://www.armory.io/privacy-policy"
 terms_of_service = "https://www.armory.io/terms-of-service"
 terms_and_conditions = "https://www.armory.io/terms-and-conditions"
 
+[params.kubesvc-plugin]
+latestVersion2="0.2.3"
+latestVersion3="0.3.13"
+latestVersion4="0.4.8"
+latestVersion5="0.5.0-rc.14"
```

Comment:
Got these versions from [this script](https://gist.github.com/nhtzr/5d20fdb2ce81ceae075831738279c894)